docs/BUILD

```diff
@@ -6,132 +6,132 @@ licenses(["notice"])
 # the --incompatible_remap_main_repo flag.
 stardoc(
     name = "build_test_docs",
-    out = "build_test_doc_gen.md",
```

Comment:
I don't see a good reason for the `out` renames here and below in docs/BUILD. "_gen" means a generated, not hand-written md doc.

Reply:
sure, I'll open a separate PR for that change and discuss my reasoning.